arch/arm64/boot/dts/freescale/crocodile.dts

```diff
@@ -181,6 +181,17 @@
 		model = "imx-audio-micfil";
 		cpu-dai = <&micfil>;
 	};
+	bat: Internal-battery {
```

Comment:
Please avoid using capital letters in node names, use `internal-battery` instead.